PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -239,7 +239,16 @@ def miniAOD_customizeCommon(process):
     process.load('RecoJets.JetProducers.QGTagger_cfi')
     task.add(process.QGTaggerTask)
 
-    process.patJets.userData.userFloats.src += [ cms.InputTag('QGTagger:qgLikelihood'), ]
+    process.patJets.userData.userFloats.src += [ 'QGTagger:qgLikelihood', ]
+
+    #HF jet shower shape
+    process.load('RecoJets.JetProducers.hfJetShowerShape_cfi')
+    task.add(process.hfJetShowerShape)
+    process.hfJetShowerShape.jets = cms.InputTag("ak4PFJetsCHS")
+    process.hfJetShowerShape.vertices = cms.InputTag("offlinePrimaryVertices")
```

Comment:
```suggestion
    process.hfJetShowerShape.jets = cms.InputTag("ak4PFJetsCHS")
    process.hfJetShowerShape.vertices = cms.InputTag("offlinePrimaryVertices")
```
not needed after the last update in `HFJetShowerShape::fillDescriptions`